spec/features/view_list_of_authors_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature "View list of Authors", type: :feature do
+  scenario "with some authors" do
+    create(:author, name: "Martin Fowler")
+    create(:author, name: "Uncle Bob")
+
+    visit authors_path
+
+    expect(page).to have_author("Martin Fowler")
+    expect(page).to have_author("Uncle Bob")
+  end
+
+  scenario "without authors" do
+    visit authors_path
+    expect(page).to have_content(I18n.t("authors.index.no_authors_found"))
```

Comment:
Just FYI, if you wish, you can skip the `I18n.` prefix, since `spec/support/i18n.rb` mixes in `AbstractController::Translation`.